modules/ROOT/pages/cache-scope-strategy.adoc

```diff
@@ -52,14 +52,26 @@ Data is immutable. This is the default value.
 +
 Data is mutable.
 
+== Synchronize Access to a Cache
+
+Mule enables you to synchronize access to a cache, which can avoid unexpected results if multiple message processors (on the same or different Mule instances) use the cache at the same time.
+
+Consider a scenario where two message processors attempt to retrieve a value from a cache but do not find the value in the cache. So each message processor calculates the value independently and inserts it into the cache. The value inserted by the second message processor overwrites the value inserted by the first message processor. If the values are different, then two different results are obtained for the same input, with only the last one stored in the cache.
+
+In some scenarios this is valid, but it can be a problem if the application requires cache coherence. Synchronizing the caching strategy ensures this coherence. A synchronized cache is locked when it is being modified by a message processor. In the previous scenario, a locked cache forces the second message processor to wait until the first message processor has calculated the value, and then it retrieves the value from the cache.
+
+Synchronization affects performance, enable this feature only if it is needed. Note that performance degradation is most severe in cluster mode.
+
+To enable synchronization, define the `synchronized` property in the `<ee:object-store-caching-strategy>` element. Accepted values are `true` and `false`.
+
 == Example Caching Strategy
 
-The following XML snippet shows the configuration of a caching strategy that defines a persistent Object Store to store the cached the responses. The caching strategy is then referenced by a Cache Scope:
+The following XML snippet shows the configuration of a caching strategy that synchronizes access to the cache, and defines a persistent Object Store to store the cached responses. The caching strategy is then referenced by a Cache Scope:
```

Comment:
"cache scope" no caps here? Or Cache scope to match the other docs (there's some weird capitalization going on in those topics). Also no caps for object store.

So is cache scope a message processor as mentioned in the scenario?